lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivationSpec.hs

```diff
@@ -211,6 +215,14 @@ spec = do
         it "XPub IcarusKey"
             (property $ prop_roundtripXPub @IcarusKey)
 
+    xdescribe "keyToHexText (not actually compatible with jcli)" $ do
```

Comment:
:thinking: ?

Reply:
Stripping the public key in `keyToHexText` was done to match our format with that of jcli. It didn't get us all the way  https://input-output-rnd.slack.com/archives/CCPKJ93Q9/p1580396136029600

You probably think we shouldn't include it as a pending test?

Reply:
But certainly it does work for keys that are rightfully generated (i.e. with the ed25519 bits tweak)? The Arbitrary instance you use in the test have encryption passphrases, so in most case, you're not trying to import a key, but it's encrypted version (which likely is an invalid key). 

This test works just fine if you use empty passphrases / unencrypted keys.

Reply:
🤨 Hmm…

Reply:
Fixed.